templates/tab/csharp/non-sso/Components/AddSSO.razor

```diff
@@ -0,0 +1,31 @@
+﻿@inject TeamsFx teamsfx
+
+<div>
+    <h2>Add Single Sign On feature to retrieve user profile</h2>
+    <p>
+        One of the advantages of building tab application in Teams is to leverage the user's identity
+        provided through Microsoft Teams (Known as
+        <a href="https://aka.ms/teamsfx-sso-doc" target="_blank" rel="noreferrer">
+            SSO support
+        </a>
+        ) and interact with Microsoft Graph for
+        building rich and seamless user experience. With Teams Toolkit, authentication is further
+        simplified by automatic AAD app registration and configuration.
+    </p>
+    <p>
+        Starting with this simple static tab app, you can follow few steps to add SSO logics to authenticate
+        users, retrieve user's profile photo.
+    </p>
+    <p>
+        See instructions (
+        <a href="https://aka.ms/teamsfx-add-sso" target="_blank" rel="noreferrer">
```

Comment:
Maybe now this link is not suitable for the VS TTK

Reply:
Not sure whether we will add new content here or use a new page. Still needs to be determined when add SSO experience is ready in VS. Will update link here if necessary.